webdriver/client.py

```diff
@@ -385,12 +377,13 @@ def send_command(self, method, uri, body=None):
 
         url = urlparse.urljoin("session/%s/" % self.session_id, uri)
         response = self.send_raw_command(method, url, body)
+        value = response.body["value"]
 
-        rv = response.body["value"]
-        if not rv:
-            rv = None
+        if response.status != 200:
```

Comment:
I think @jgraham’s idea of calling `session.transport.send` is enticing. I don’t like adding a new kwarg because it increases the complexity. I also think it’s acceptable to make `send_command` cleverer because it is associated with the `Session` object, which already knows if a session exists or not.